core/templates/dev/head/components/ProfileLinkImageDirective.js

```diff
@@ -32,18 +32,21 @@ oppia.directive('profileLinkImage', [function() {
         return GLOBALS.SYSTEM_USERNAMES.indexOf(username) == -1;
       };
 
-      $scope.profileImageUrl = (
-        '/preferenceshandler/profile_picture_by_username/' + $scope.username());
-      $scope.profilePicture = DEFAULT_PROFILE_IMAGE_PATH;
+      if ($scope.username() !== 'Community Owned') {
```

Comment:
Hey @seanlip , I have created a new directive by the name circularImage which takes in an image and an optional link. But I just realized, in case of the "Community Owned' , we cannot provide the image link since the globe icon that is currently being displayed is a font-awesome icon, not an image.

In such a case, since both the image and the link are unavailable, it renders the directive to be useless! How do I get around this problem ?

Thanks.